CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Upgrade to latest helm and k8s client dependencies (https://github.com/pulumi/pulumi-kubernetes/pulls/2292)
```

Comment:
It should be `pull/2292`, not `pulls/2292`

Reply:
Thank you! Opened #2310.